rules-reviewed/quarkus/springboot/springboot-di-to-quarkus.windup.xml

```diff
@@ -30,5 +30,24 @@
                 </hint>
             </perform>
         </rule>
+        <rule id="springboot-di-to-quarkus-00001">
+            <when>
+                <or>
+                    <xmlfile matches="//*/b:bean/@class">
+                        <namespace prefix="b" uri="http://www.springframework.org/schema/beans"/>
+                    </xmlfile>
+                    <xmlfile matches="//*/c:annotation-config">
+                        <namespace prefix="c" uri="http://www.springframework.org/schema/context"/>
+                    </xmlfile>
+                </or>
+            </when>
+            <perform>
+                <hint title="XML-based and annotation-based bean configuration metadata is not supported by Quarkus " effort="1" category-id="mandatory">
+                    <message>Quarkus only supports the java-based configuration of Spring beans.</message>
```

Comment:
I think we need to make clear in the title and message that this is specifically for IoC/DI containers only (Quarkus does support these other kinds of configuration metadata in other contexts)

Reply:
Good point Mark - I'll update the message.